packages/composite-checkout/src/components/checkout-review-order.js

```diff
@@ -65,3 +67,91 @@ LineItem.propTypes = {
 		} ),
 	} ),
 };
+
+function TermsAndConditions() {
+	const isDomainsTermsVisible = useHasDomainsInCart();
+
+	return (
+		<TermsAndConditionsWrapper>
+			<TermsParagraph>
+				<strong>By checking out:</strong> you agree to our{' '}
```

Comment:
@sirbrillig any ideas how we might localize all this copy?

Reply:
Good question. I think that our localization function will dictate how we mix HTML and text. `@wordpress/i18n` has [an `sprintf` function](https://github.com/WordPress/gutenberg/blob/master/packages/i18n/README.md) that could handle the separation. Calypso uses [an interpolated format](https://github.com/Automattic/wp-calypso/blob/master/packages/i18n-calypso/README.md#mixing-strings-and-markup).

Why don't you change this for now to resemble the Calypso way:

```js
localize( '{{strong}}By checking out:{{/strong}} you agree to our {{tosLink}}Terms of Service{{tosLink}}', 
  {
    components: {
      strong: <strong />, 
      tosLink: <a href="https://wordpress.com/tos/" /> 
    } 
} );
```